docs/2_4_common_schema.adoc

```diff
@@ -855,1491 +807,1469 @@ Element `<fmiModelDescription><DefaultExperiment>` is defined as:
+|`causality`
+|
+[[causality,`causality`]]
+Enumeration that defines the causality of the variable.
+Allowed values of this enumeration:
 
-====== Icon
+[[parameter,`parameter`]]
+`= parameter`: A data value that is constant during the simulation (except for <<tunable>> parameters, see there) and is provided by the environment and cannot be used in connections, except for parameter propagation in terminals as described in <<GeneralRemarkOnSignal>>.
+<<variability>> must be <<fixed>> or <<tunable>>.
+These parameters can be changed independently, unlike <<calculatedParameter,calculated parameters>>.
+<<initial>> must be <<exact>> or not present (meaning <<exact>>).
 
-.Icon element.
-[[figure-schema-Icon]]
-image::images/schema/Icon.png[width=40%, align="center"]
+[[calculatedParameter,`calculatedParameter`]]
+`= calculatedParameter`: A data value that is constant during the simulation and is computed during initialization or when <<tunable>> <<parameter,`parameters`>> change.
+<<variability>> must be <<fixed>> or <<tunable>>.
+<<initial>> must be <<approx>>, <<calculated>> or not present (meaning <<calculated>>).
 
-The extent and position of the FMU icon are defined in the `<Icon>` element.
-The optional image file of the FMU icon is placed at the path `icons/icon.png` in the ZIP archive of the FMU.
-The terminals should not be visible in the image.
-Optionally an SVG file can be provided if also the PNG file is present.
-This enables high quality rendering and printing in importing tools.
-This SVG file has to be placed at the path `icons/icon.svg` in the ZIP archive.
+[[input,`input`]]
+`= input`: The variable value can be provided by the importer. +
+_[For example, the importer could forward the output of another FMU into this input.]_
 
-The point (`x1`, `y1`) maps to the left lower corner of the PNG image or SVG viewport.
-The point (`x2`, `y2`) maps to the right upper corner of the PNG image or SVG viewport.
+[[output,`output`]]
+`= output`: The variable value can be used by the importer. +
+_[For example, this value can be forwarded to an input of another FMU.]_ +
+The algebraic relationship to the <<input,`inputs`>> can be defined via the <<dependencies>> attribute of <<Output,`<fmiModelDescription><ModelStructure><Output>`>>.
 
-====== Placement, Extent, and Painting Order of Graphical Items
+[[local,`local`]]
+`= local`: Local variables are: +
 
-image::images/GraphicalRepresentation.svg[width=70%, pdfwidth=70%, align="center"]
+* continuous <<state,`states`>> and their <<StateDerivative,`StateDerivative`pass:[s]>>, <<DiscreteTimeState,`DiscreteTimeState`pass:[s]>>, <<EventIndicator,`EventIndicator`pass:[s]>> or <<InitialUnknown,`InitialUnknown`pass:[s]>>.
+These variables are listed in the `<fmiModelDescription><ModelStructure>`. +
+* [[localClock,`local clock`]] internal, intermediate variables or local clocks which can be read for debugging purposes and are not listed in the `<fmiModelDescription><ModelStructure>`.
```

Comment:
Do local clocks still exist?

Reply:
We have tried to keep clocks a "normal" variable.
So, yes, local clocks are not forbidden in the current document.